src/CassandraChangeFeedSample/Program.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;

```

Comment:
nit: If possible, add copyright, like https://github.com/Azure-Samples/cosmos-dotnet-functions-migration/blob/master/Src/MigrationExecutorFunctionApp/Startup.cs#L1

Reply:
This on any .cs file